.pre-commit-config.yaml

```diff
@@ -20,5 +20,13 @@ repos:
 - repo: https://github.com/pre-commit/pre-commit-hooks
   rev: v4.3.0
   hooks:
+  - id: check-merge-conflict
   - id: check-yaml
   - id: requirements-txt-fixer
+  - id: trailing-whitespace
```

Comment:
We should consider adding thse two to our templates repo.

Reply:
Done in https://github.com/ansys/ansys-templates/pull/240.